### DISCUSSION TITLE
Anyone build API over this libraray?

### DISCUSSION THREAD

**Discussion by rajuaryan21** [tulir/whatsmeow#192] [General]
Did anyone manage to build API using this library? Please, let me know. I would like to test it. I can't build this API myself as I am not a go developer.

**Comment by sumnerevans** [comment]
The https://github.com/mautrix/whatsapp/ project uses this library.

**Comment by abbasudo** [comment]
@rajuaryan21 I did. I modified the `mdfile` file and built an executable file out of it. then called it within a laravel project and built an API.

**Comment by hrizal** [comment]
i think everyone come to here build api using this lib 😁
